crates/nil/src/server.rs

```diff
@@ -163,7 +163,20 @@ impl Server {
             None => std::env::current_dir().expect("Failed to the current directory"),
         };
 
-        *Arc::get_mut(&mut self.config).expect("No concurrent access yet") = Config::new(root_path);
+        *Arc::get_mut(&mut self.config).expect("No concurrent access yet") = {
```

Comment:
Please unwrap the block and move the Arc setting line below.

Reply:
done